python_modules/dagster-graphql/dagster_graphql/implementation/fetch_runs.py

```diff
@@ -106,6 +107,44 @@ def get_runs(graphene_info, filters, cursor=None, limit=None):
     return [GrapheneRun(run) for run in runs]
 
 
+def get_in_progress_runs(graphene_info):
+    instance = graphene_info.context.instance
+
+    in_progress_runs_filter = PipelineRunsFilter(
+        statuses=[
+            PipelineRunStatus.STARTING,
+            PipelineRunStatus.MANAGED,
+            PipelineRunStatus.NOT_STARTED,
+            PipelineRunStatus.QUEUED,
+            PipelineRunStatus.STARTED,
+            PipelineRunStatus.CANCELING,
+        ],
+    )
+
+    return instance.get_runs(in_progress_runs_filter)
```

Comment:
I don't think this is quite right... I don't think we should query runs without a `pipeline_name` filter.  For one, we might have a different repository that has a pipeline with the same step name.

Reply:
@prha thanks, this makes sense. Updated to search only for runs of jobs in the repo.